Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM php:apache
+
+RUN apt-get update && apt-get install -y \
+        libfreetype6-dev \
+        libjpeg62-turbo-dev \
+        libpng12-dev \
+        wget \
+        zip \
+        unzip; \
+    # We install and enable php-gd
+    docker-php-ext-configure gd --with-freetype-dir=/usr/include/ --with-jpeg-dir=/usr/include/; \
```

Comment:
You might also want to install mycrypt (or Libsodium) for the random number generator. But if read access to /dev/urandom is guaranteed, then this is of course not necessary.